public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+App.controller('App.Ctrl', function($scope, $timeout, AppService) {
+    const colorCode = {1:"green", 2: "red", 3: "yellow", 4: "blue"};
+    const speed = 500; // in milliseconds
+    var on = false;
+    var lock = false;
+    var strict = false;
+    var colorSequence = [];
+    var count = 0;
+    var round = 1;
+    var turnOffTimer;
+    var recursiveTimer;
+    var timer;
+    var sound = {1:{}, 2:{}, 3: {}, 4: {}};
+    var buzzer;
+    var win;
+
+    $scope.init = function() {
+        if(on === true) {
+            setUpSoundEffects();
+            reset(true);
+        }
+    };
+
+    $scope.toggleOnOff = function() {
+        if(on === true) {
+            on = false;
+            $scope.btn__off = {"background-color": "#8A96B6"};
+            $scope.btn__on = {"background-color": "#333"};
+            cancelAllTimers();
+            turnOffAllLights();
+            $scope.screen = "";
+        }
+        else {
+            on = true;
+            $scope.btn__off = {"background-color": "#333"};
+            $scope.btn__on = {"background-color": "#8A96B6"};
+        }
+    };
+
```

Comment:
Used your booleanValue = !booleanValue expression in ng-click to toggle the strict button.  I kept the $scope.toggleOnOff function though because I need other functions to execute (i.e. cancelAllTimers(), turnOffAllLights() ).